src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -22,17 +22,31 @@ $side-nav-color-dark: #323a45;
     cursor: pointer;
     width: calc(100% - 45px);
     margin-left: 22px;
+
+    &::after {
+      content: "\A";
```

Comment:
Pardon my ignorance, but what does `"\A"` do? I'm assuming that's not rendering an "A"

Reply:
Adds a line break without having to add a `<br>`. See https://stackoverflow.com/questions/34060665/css-content-character-a-takes-more-space-than-br-tag/34060752